### ISSUE TITLE
TypeScript Autosuggest not working after file creation in Remote-Containers Project on WSL2

### ISSUE THREAD

**Issue by yngwi** [microsoft/vscode#98799]
- VSCode Version: 1.45.1
- OS Version: Windows 10 2004 Build 19041.264 (Windows Insiders Release Preview)
- Latest Remote-Containers version: 0.117.1
- Docker Container: node:latest (Debian stretch)

Steps to Reproduce:

1. Start Remote-Containers Project inside Docker for Windows with WSL2
2. Create new App with create-react-app and the typescript template
3. Create Component in new file
4. Autosuggest stops working (Probably because Typescript Language Server crashes)

Does this issue occur when all extensions are disabled?: Yes/No
No because it is in relation with the remote-containers extension

The following part from the Remote Extension Log seems relevant:

```
[2020-05-29 09:47:03.022] [exthost] [error] [vscode.typescript-language-features] provider FAILED
[2020-05-29 09:47:03.023] [exthost] [error] Error: DisposableStore is missing...
	at U._convertCompletionItem (/root/.vscode-server/bin/5763d909d5f12fe19f215cbfdd29a91c0fa9208a/out/vs/server/remoteExtensionHostProcess.js:698:574)
	at U.resolveCompletionItem (/root/.vscode-server/bin/5763d909d5f12fe19f215cbfdd29a91c0fa9208a/out/vs/server/remoteExtensionHostProcess.js:698:354)
	at processTicksAndRejections (internal/process/task_queues.js:89:5)
[2020-05-29 09:47:03.267] [exthost] [info] [vscode.typescript-language-features] UNSAVE result from 'resolveCompletionItem', extension SHOULD NOT change any of: additionalTextEdits, or command
```

**Comment by mjbvz** [comment]
Do you also see this in VS Code 1.46? I just tested on wsl2 but can't reproduce the issue

**Comment by yngwi** [comment]
Hi @mjbvz,
I'm sorry for my late reply. I just tried using 1.46 and it seems different but still doesn't work as expected.

In my Create-React-App with TypeScript / Remote Container I just added a new folder for a Component, added a new index.tsx file for the component code and added code for the component. After saving I went to another file and started to add the Component, on CTRL-Space the component doesn't appear in the autocomplete list. After I restart the TS server, the Component appears. However, now the pre-existing components appear just fine in autocomplete. Also, the error I've mentioned above doesn't appear anymore. So maybe now it's something in my system?

Regards,
Daniel

**Comment by mjbvz** [comment]
Does the same thing work in non-remote workspace? I'm still having issues reproducing this so I want to understand if this the problem is related to wsl2 or to the project structure

**Comment by yngwi** [comment]
Hi @mjbvz,

sorry for being so slow but I had some busy weeks and couldn't further work on this issue. I hope it can be re-opened. I have now tested some more:

Prerequisites:

- Windows 10 Version 10.0.19041 Build 19041
- VSCode Insider 1.48.0 (Build from 2020-07-10) (fresh install, only Remote Containers Extension installed)
- Remote Containers v0.128.0
- Node on Windows 14.5.10

I created the following React test apps:

1. `npx create-react-app test1`
2. `npx create-react-app test2` opened in Node Container
3. `npx create-react-app test3 --template typescript`
4. `npx create-react-app test4 --template typescript` opened in Node Container

For each of them I created a Component called "Test" in a new file in the `src` folder (`Test.jsx` or `Test.tsx`):

```
import React from "react";
const Test = () => <div>test</div>;
export default Test;
```

I then tried to add the new component in the main App Component by writing `<Test` and pressing `ctrl+space` to trigger the auto-completion and automatic insert of the import statement.

The process was successful for all apps except Number 4 (TS in Container), in that one the Component was only autocompleted after issuing the `Typescript: Restart TS Server`. For me this is reproducable every time.

The log doesn't seem to display anything related.
